kubernetes/PostTarget.cmake

```diff
@@ -0,0 +1,9 @@
+target_link_libraries(${pkgName} yaml websockets)
+set_target_properties(${pkgName} PROPERTIES LINKER_LANGUAGE C)
```

Comment:
What's the reason that `LINKER_LANGUAGE` is set to `C` here ?

Reply:
Linking to this library from a C++ project did not work for me without this. I think that it is related to name mangling.